garrysmod/gamemodes/terrortown/gamemode/lang/french.lua

```diff
@@ -0,0 +1,1084 @@
+---- French language strings
+
+local L = LANG.CreateLanguage("Français")
+
+--- General text used in various places
+L.traitor    = "Traitre"
+L.detective  = "Détective"
+L.innocent   = "Innocent"
+L.last_words = "Derniers mots"
+
+L.terrorists = "Terroristes"
+L.spectators = "Spectateurs"
+
+--- Round statut messages
+L.round_minplayers = "Pas assez de joueurs pour commencer un nouveau round..."
+L.round_voting     = "Vote en cours, le round est retardé de {num} secondes..."
+L.round_begintime  = "Un nouveau round commence dans {num} secondes. Préparez-vous."
+L.round_selected   = "Les Traitres ont été choisis."
+L.round_started    = "Le round a commencé!"
+L.round_restart    = "Le round a recommencé de force par un admin."
+
+L.round_traitors_one  = "Traitre, vous êtes seul."
+L.round_traitors_more = "Traitre, voici vos alliés: {names}"
+
+L.win_time         = "Temps écoulé. Les Traitres ont perdu."
+L.win_traitor      = "Les Traitres ont gagné!"
+L.win_innocent     = "Les Traitres ont été battus!"
+L.win_showreport   = "Regardons le rapport du round {num} secondes."
+
+L.limit_round      = "Limite de round atteinte. {mapname} va bientôt charger."
+L.limit_time       = "Limite de temps atteinte. {mapname} va bientôt charger."
+L.limit_left       = "Il reste {num} round(s) ou {time} minutes avant que la carte change vers {mapname}."
+
+--- Credit awards
+L.credit_det_all   = "Détectives, vous avez reçu {num} crédit(s) d'équipement pour votre performance."
+L.credit_tr_all    = "Traitres, vous avez reçu {num} crédit(s) d'équipement pour votre performance."
+
+L.credit_kill      = "Vous avez reçu {num} crédit(s) pour avoir tué un {role}."
+
+--- Karma
+L.karma_dmg_full   = "Votre Karma est à {amount}, donc vous ferez plein dégâts ce round!"
+L.karma_dmg_other  = "Votre Karma est à {amount}. De ce fait tous les dégâts que vous faites seront réduits de {num}%"
+
+--- Body identification messages
+L.body_found       = "{finder} à trouvé le corps de {victim}. {role}"
+
+-- The {role} in body_found will be replaced by one of the following:
+L.body_found_t     = "C'était un Traitre!"
+L.body_found_d     = "C'était un Détective."
+L.body_found_i     = "C'était un Innocent."
+
+L.body_confirm     = "{finder} a confirmé la mort de {victim}."
+
+L.body_call        = "{player} a appelé un Détective pour le corps de {victim}!"

```

Comment:
Add a space before the `!`

Reply:
All :, !, and ? now have a space in front of them